mycroft/res/ui/settings/ssh_settings.qml

```diff
@@ -88,7 +88,7 @@ Item {
                 text: qsTr("Disable SSH")
                 onClicked: {
                     Mycroft.SoundEffects.playClickedSound(Qt.resolvedUrl("../../snd/clicked.wav"))
-                    triggerGuiEvent("mycroft.device.set.ssh", {"enable_ssh": false})
+                    triggerGuiEvent("system.ssh.disable", {"display": false})
```

Comment:
is this correct? should it be a regular bus message instead of GuiEvent?

Reply:
fixed